client/src/Components/Activities/Activities.css

```diff
@@ -16,14 +17,14 @@ h2 {
 
 .activity-card {
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
   align-items: center;
-  height: 300px;
-  width: 40%;
+  text-align: center;
```

Comment:
YES

Reply:
Center align looks great for the activity cards!